### ISSUE TITLE
More informative info regarding goto-definition

### ISSUE THREAD

**Issue by azzamsa** [jorgenschaefer/elpy#1719]
My  (elpy-goto-definition) always failing couple of months now. But I don't have time to fix it yet until today. I've tried to setup simple project to see where elpy-goto-definition failing:

it didn't work even just two file in the same directory with existing `.git` and `__init__.py` repo. After digging for while, I find [Chillar recommends to use jedi as rpc](https://github.com/jorgenschaefer/elpy/issues/948#issuecomment-243744062), turns out I've add that line year ago (that's why goto-definition works in my other laptop).

Turn out jedi is not installed in my current venv. after doing `pip install jedi` it works well (even I've tried with project that consist of 1199 files).

**TLDR**;

Can we provide useful message if rpc-backend not configured properly? so user don't accuse of elpy failing.

And, do we need to add a link / more description [here](https://elpy.readthedocs.io/en/latest/ide.html#command-elpy-goto-definition)

Thank you.

**Comment by galaunay** [comment]
I added an error message when trying to use jedi related functions without jedi installed (#1723).

The idea is to redirect users to `elpy-config` that provides a good overview of the state of Elpy.

Does it seem sensible ?

**Comment by azzamsa** [comment]
```
 (error "This feature requires the `jedi` package to be installed. Please check `elpy-config` for more information.")))
```

Yes, looks better :)

**Comment by galaunay** [comment]
Just merged it now, 

Thanks for the feedback !
